src/components/Header.jsx

```diff
@@ -55,15 +55,23 @@ export default function Header({username}) {
     <div>
       <Toolbar />
       <List>
+      <ListItem key='home'>
+      <Link to={`/`}>
+        <ListItemText primary={<Typography align="center">Home</Typography>} />
+        </Link>
+        </ListItem>
+        <Divider />
         <ListItem key='heading'>
         <ListItemText primary={<Typography align="center">Topic</Typography>} />
         </ListItem>
         <Divider />
         {topics.map((topic, index) => (
           <ListItem sx={{textAlign: 'center'}}key={index} disablePadding>
```

Comment:
would be better to use topic.slug as the unique key instead of index to avoid future errors